### PR TITLE
Added !status

### PR DESCRIPTION
**Pull request by LW001**
As described in https://trello.com/c/Do2gYoM4/.

Using the command works the following way:
`!status <ID/LINK> <NEW STATUS> <COMMENT>`
* ID/LINK works using the magical UVRegex(TM)
* NEW STATUS accepts the following values:
`review, re, under review, ur, u, planned, plan, pl, p, started, start, completed, complete, co, c, declined, decline, de, d`
* COMMENT accepts a comment to be added, also filtering out pipes.

UV endpoint: https://developer.uservoice.com/docs/api/reference/#_api_v1_forums_forum_id_suggestions_suggestion_id_respond_put

I actually ran a syntax check this time, are you proud yet

### CODE REVIEW

File: Commands/Commands/uservoice.js [hunk 1]
```diff
@@ -207,6 +207,79 @@ commands.comment = {
...
+      uvClient.v1.loginAs(email).then(c => { // !status id msg
+        var grabidregex = /(under review|review|re|ur|u|planned|plan|pl|p|started|start|completed|complete|co|c|declined|decline|de|d)+(.{1,})/
+        let s = suffix.split(' ')
+        let idt = s[0]
+        s.shift()
+        let statncontent = s.join(' ')
+        let grabstatusidfromhere = statncontent.match(grabidregex)
+        let statusname = grabstatusidfromhere[1]
+        let statusid
+        if (statusname === 'review' || statusname === 're' || statusname === 'under review' || statusname === 'ur' || statusname === 'u') statusid = '1707873'
```
Comment (Dougley) [inline Commands/Commands/uservoice.js]: Use a regex

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/uservoice.js [hunk 2]
```diff
@@ -219,14 +219,13 @@ commands.status = {
         let idt = s[0]
         s.shift()
         let statncontent = s.join(' ')
-        let grabstatusidfromhere = statncontent.match(grabidregex)
+        let grabstatusidfromhere = statncontent.toLowercase().match(grabidregex)
```
Comment (SteamingMutt) [inline Commands/Commands/uservoice.js]: Use a case insensitive regex instead

File: Commands/Commands/uservoice.js [hunk 3]
```diff
@@ -207,6 +207,78 @@ commands.comment = {
...
+  fn: function (bot, msg, suffix, uvClient, cBack) {
+    msg.channel.sendTyping()
+    getMail(uvClient, msg.author.id).then(email => {
+      uvClient.v1.loginAs(email).then(c => { // !status id msg
+        var grabidregex = /(under review|review|re|ur|u|planned|plan|pl|p|started|start|completed|complete|co|c|declined|decline|de|d)+(.{1,})/
+        let s = suffix.split(' ')
+        let idt = s[0]
+        s.shift()
+        let statncontent = s.join(' ')
+        let grabstatusidfromhere = statncontent.toLowercase().match(grabidregex)
```
Comment (Dougley) [inline Commands/Commands/uservoice.js]: Use a case insensitive regex instead

**Review by Dougley — CHANGES_REQUESTED** [review]
Last nitpick before I'll merge this

**Comment by LW001** [comment]
I fucked this one up quite a ton, cant edit it anymore because I deleted the repo, trying to get that fixed lul

**Comment by LW001** [comment]
Kinda won't be able to recover that repo, http://imgur.com/a/9CiLZ

Closing PR and reopening it later today, sorry for the inconvenience.

**Pull request closed without merging**